File: abc/abc179/abc179_c.cs
// https://atcoder.jp/contests/abc179/submissions/16892897

using System;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var result = 0;
        for (int i = 1; i < N; i++)
        {
            result += (N - 1) / i;
        }
        Console.WriteLine(result);
    }
}

File: abc/abc179/abc179_c.fs
// https://atcoder.jp/contests/abc179/submissions/16893345

let N = stdin.ReadLine() |> int

let rec solver r i =
    if i = N then r
    else solver (r + (N - 1) / i) (i + 1)

solver 0 1 |> stdout.WriteLine
